workflows-scripts/users/users-loan-renewal/README.md

```diff
@@ -0,0 +1,36 @@
+## Workflow steps:
```

Comment:
This readme is specific to loan renewal. Don't need it.

Reply:
Argh, it did check in the users-loan-renewal files. Let me try to back them out.

Reply:
Removed users-loan-renewal files.